Blizzard.IO.Core/IDeserializer.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Blizzard.IO.Core
+{
+    public interface IDeserializer<TData>
+    {
+        TData Desirialize(byte[] data);
```

Comment:
you have a type it is Deserialize
```suggestion
        TData Deserialize(byte[] data);
```

Reply:
f